config/load.go

```diff
@@ -116,6 +120,38 @@ func (c *Config) checkConfig() error {
 	return nil
 }
 
+// Return Consul configuration from environment variables
+func (c *Config) setEnvConsulConfig() {
```

Comment:
Would it make sense to log a message if we load something from ENV?

Reply:
Do you mean if we load anything from ENV we log a single message, or log a message per ENV variable loaded? 

I would imagine it would be more towards the latter, with a debug message also showing the actual value from the ENV variable that was used. 

But I'm very inexperienced so I am open to being completely wrong about that!

Reply:
The beauty is: it's completely up to us on how to go about this! ;-)
I personally prefer to log per variable. There are not a lot of them, so shouldn't be a huge amount of logs that potentially get generated.

Reply:
Ha, fair :)

Thanks for the help, working on it now!